### ISSUE TITLE
Is there a planned payment mechanism for storing cxo?

### ISSUE THREAD

**Issue by steady-clackin** [skycoin/cxo#143]
I could see such a mechanism powering basically a global network file store. I.e. something really really useful. It could come with replication controls so that the owner can choose the desired replication (paying whatever the rate is for the amount of data that results). Pay more to have more guarantee that the data won't be lost, etc. In this manner skycoin could not only be asset-backed by bandwidth capacity but also asset-backed by storage capacity.

**Comment by steady-clackin** [comment]
I think such a system would incentize content hosting on the network, since you hosts would be compensated both for storing and serving it (the latter being true only if i understand the bandwidth compensation system correctly, not 100% sure tho).

**Comment by logrusorgru** [comment]
Yes. That is what SPACO is doing.

https://spaco.online/2017/12/10/big-news-the-announcement-about-strategic-partnership-between-skycoin-and-spaco/

**Comment by steady-clackin** [comment]
neat. is there any more info besides a marketing release? :)

**Comment by logrusorgru** [comment]
No, there is not. I can't help. The SPARCO is separate project. Sorry.
